### ISSUE TITLE
Gem for 0.2.9 has a broken link

### ISSUE THREAD

**Issue by SuperQ** [davishmcclurg/json_schemer#60]
When trying to use 0.2.9, the Gem unpacks with a broken symlink:

```
lrwxr-xr-x wheel/wheel       0 2020-02-01 19:06 JSON-Schema-Test-Suite/tests/latest -> JSON-Schema-Test-Suite/tests/draft2019-09
```

This is causing my rspec to fail in weird ways:
```
$ bundle exec rspec

An error occurred in a `before(:suite)` hook.
Failure/Error:
  File.open(path()) do |s|
    File.open(dest, 'wb', s.stat.mode) do |f|
      IO.copy_stream(s, f)
    end
  end

Errno::ENOENT:
  No such file or directory @ rb_sysopen - /home/ben/gitlab.com/gitlab-cookbooks/gitlab-exporters/.bundle/gems/json_schemer-0.2.9/JSON-Schema-Test-Suite/tests/latest
# ./.bundle/gems/berkshelf-7.0.9/lib/berkshelf/berksfile.rb:667:in `block (3 levels) in vendor'
```

**Comment by davishmcclurg** [comment]
Fixed in 0.2.10. Thanks @SuperQ!

**Comment by SuperQ** [comment]
Thanks!
